.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.0', '3.1', '3.2']
+        ruby-version: ['3.2']
         ES_VERSION: ['5.6.15', '8.7.0']
         include:
           - ES_VERSION: '5.6.15'
```

Comment:
Does this mean that we're no longer supporting 3.0 or 3.1 at all with elastomer client? Will that cause any customer impact, if users are depending upon that support?

Reply:
Yeah, since the library is only used internally, we're focusing support on the version we run.